shell/pages/c/_cluster/apps/charts/index.vue

```diff
@@ -501,16 +493,72 @@ export default {
           <h1>{{ t('catalog.charts.noCharts') }}</h1>
         </div>
       </div>
-      <SelectIconGrid
-        v-else
-        data-testid="chart-selection-grid"
-        component-test-id="chart-selection"
-        :rows="filteredCharts"
-        name-field="chartNameDisplay"
-        description-field="chartDescription"
-        :color-for="colorForChart"
-        @clicked="(row) => selectChart(row)"
-      />
+      <template v-else>
+        <div
+          class="app-cards"
+          data-testid="app-cards-container"
+        >
+          <ItemCard
+            v-for="chart in filteredCharts"
+            :id="chart.chartName"
+            :key="chart.chartName"
+            variant="medium"
+            :pill="chart.featured ? { label: { key: 'generic.shortFeatured' }, tooltip: { key: 'generic.featured' }} : undefined"
+            :header="{
+              title: { text: chart.chartNameDisplay },
+              statuses: chart.cardContent.statuses
+            }"
+            :image="{ src: chart.versions[0].icon, alt: { text: t('catalog.charts.iconAlt', { app: get(chart, 'chartNameDisplay') }) }}"
+            :content="{ text: chart.chartDescription }"
+            :value="chart"
+            :clickable="true"
+            @card-click="selectChart"
+          >
+            <template #item-card-sub-header>
+              <div class="app-card-sub-header">
+                <div
+                  v-for="subHeaderItem in chart.cardContent.subHeaderItems"
+                  :key="subHeaderItem.text"
+                  class="app-card-sub-header-item"
+                  data-testid="app-card-version"
+                >
+                  <i
+                    v-clean-tooltip="t(subHeaderItem.iconTooltip.key)"
+                    :class="['icon', 'app-card-item-icon', subHeaderItem.icon]"
+                  />
+                  <p>
+                    {{ subHeaderItem.label }}
+                  </p>
+                </div>
+              </div>
+            </template>
+            <template #item-card-footer>
+              <div class="app-card-footer">
```

Comment:
These are underlined which dissappears on hover, which is the behaviour of link... but they're not links. what was the long term plan for these, to be shortcuts to applying them as filters?